src/scss/lexicon-base/_nav-tabs.scss

```diff
@@ -34,7 +34,7 @@
 	}
 }
 
-@media screen and (max-width: $grid-float-breakpoint-max) {
```

Comment:
I couldn't find a replacement for the ```$grid-float-breakpoint-max``` variable. Is there a better option to use that I can use for future component updates?